cmd/goal/node.go

```diff
@@ -61,7 +61,7 @@ var newNodeFullConfig bool
 var watchMillisecond uint64
 var abortCatchup bool
 var fastCatchupForce bool
-var initializeCatchup uint64
+var minRounds uint64
```

Comment:
```suggestion
var minCatchupRounds uint64
```

Reply:
Thanks Bob, I renamed the variable to be clearer.

All the globals in these old goal commands really bug me. I've done away with them in the other commands I work on: https://github.com/algorand/go-algorand/blob/master/tools/block-generator/runner/runner.go#L30

Reply:
Yeah that is a better way to go but meh.